### PR TITLE
Implement dialogue-normalization Skill

### LINKED ISSUE #44: [Feature] Implement the dialogue-normalization Skill
## User need

Codex needs to normalize dialogue loudness safely through the generic Framekit Skill lifecycle, using measured audio evidence before and after the edit.

## Proposed behavior

Register a versioned `dialogue-normalization` Skill that operates on complete clip occurrences.

The Skill performs:

1. Capability resolution
2. Revision-bound target and occurrence resolution
3. Dialogue and loudness measurement
4. Deterministic gain planning
5. Non-mutating preview
6. Composite `set-gain` execution
7. Canonical read-after-write
8. New post-write loudness measurement
9. LUFS and true-peak verification
10. Accept or rollback

Inputs include or resolve versioned defaults for:

- Target LUFS
- Loudness tolerance
- Maximum true peak
- Minimum and maximum gain
- Minimum valid dialogue duration

### In scope

- Versioned `dialogue-normalization` Skill
- Complete clip-occurrence targeting
- Target and safety policy inputs
- Measurement and gain-plan preview
- Composite clip-level `set-gain`
- Post-write re-measurement
- LUFS and true-peak verification
- Complete rollback
- Generic MCP Skill invocation examples

### Out of scope

- Partial-clip gain keyframes
- Compression or limiting
- Noise reduction
- Music mixing
- Speaker diarization and balancing
- Role-based final mix
- Persistent execution replay

## Safety and capability boundaries

The Skill requires:

- Canonical timeline read
- Dialogue detection
- Range-bound loudness and true-peak analysis
- Supported clip-level `set-gain`
- Timeline write
- Read-after-write
- Composite transaction rollback

No dialogue, invalid measurement, insufficient duration, excessive gain, or peak risk must produce a non-mutating `SKIP`.

Predicted post-gain values cannot satisfy verification. The Skill must perform a new measurement after the write.

## Dependencies

- Depends on #30 for composite transactions.
- Depends on #32 for live canonical Final Cut reads, `set-gain`, read-after-write, and rollback.
- Depends on #33–#37 for the Skill lifecycle and MCP surface.
- Depends on #41 for range-bound loudness analysis and dialogue gain planning.

## Acceptance criteria

- [ ] The Skill is discoverable through `skill.list` and `skill.inspect`.
- [ ] Missing requirements make the Skill unavailable before planning.
- [ ] Preview reports occurrence, measured LUFS, true peak, target, tolerance, gain, estimated peak, decision, warnings, and token.
- [ ] Preview does not mutate editor state.
- [ ] Execute applies only the preview-authorized `set-gain` operations.
- [ ] Clips already inside tolerance return a verified no-op.
- [ ] No dialogue, silence, insufficient duration, gain-clamp violation, and peak risk do not mutate state.
- [ ] Post-write verification uses a new measurement.
- [ ] Final LUFS is inside tolerance.
- [ ] Final true peak is at or below the configured limit.
- [ ] The canonical diff contains only authorized gain changes.
- [ ] Any failed operation, measurement, or verification check rolls back the complete transaction.
- [ ] The Skill contains no Final Cut-specific commands.

## Validation

Add runtime and MCP integration tests for quiet dialogue, loud dialogue, multiple clips, already-normalized clips, no dialogue, silence, insufficient duration, gain-clamp violation, peak risk, stale context, invalid token, post-write measurement failure, LUFS verification failure, peak verification failure, unexpected diff, and rollback.

Add deterministic FCPXML coverage for clip-level gain and opt-in live Final Cut validation through #32.

### PR DESCRIPTION
**Pull request by morshoto**
## Summary

- Implement the versioned generic `dialogue-normalization` Skill for complete clip occurrences.
- Add default policy normalization, source-range-bound audio measurement, deterministic gain planning, and explicit APPLY/NO_OP/SKIP decisions.
- Enforce post-write remeasurement, LUFS/true-peak verification, authorized-diff checks, and canonical rollback on failure.
- Advertise `set-gain` in the deterministic FCPXML adapter and document the Skill contract.

## Validation

- `pnpm install --frozen-lockfile`
- `pnpm run build`
- `pnpm run test` — 550 passed
- `pnpm run check:boundaries`
- Native Swift/Xcode gates skipped: no native files changed.

## TDD coverage

The committed test steps cover defaults and discovery, repeated occurrences, source ranges, no-op and safety skips, gain clamping, peak risk, stale and invalid measurements, post-write failures, unauthorized diffs, rollback, MCP normalization, FCPXML gain output, and documentation.

Closes #44

### CODE REVIEW

File: packages/runtime/src/skills/builtin.ts [hunk 1]
```diff
@@ -334,13 +352,19 @@ async function planDialogueSkill(context: SkillPlanningContext, input: Record<st
       gainDb: plan.clampedGainDb,
```
Comment (morshoto) [inline packages/runtime/src/skills/builtin.ts]: [must] `set-gain` is an absolute clip value in both the in-memory and FCPXML adapters, but `planDialogueGain` returns only the desired delta (`targetLufs - measuredLufs`) and this passes that delta directly as `gainDb`. For a clip already at +2 dB and measuring -18 LUFS, the preview emits `gainDb: 2`, leaves the effective gain unchanged, and the post-write check rolls back instead of normalizing to -16 LUFS. Compute the absolute target from the existing occurrence gain while preserving the bounded change policy, and add a regression test with a non-zero pre-existing gain.

**Review by morshoto — COMMENTED** [review]
[must] Please resolve the absolute-vs-delta gain mismatch called out in the inline comment.

File: packages/runtime/src/skills/builtin.ts (hunk 1 above)
Comment (morshoto) [reply packages/runtime/src/skills/builtin.ts]: [fixed] Applied in 98a6070. The planner preserves the bounded delta for policy and diagnostics, then emits the absolute clip target as existingGainDb plus clampedGainDb. Added a regression for a clip at +2 dB: preview emits +4 dB and execute verifies with gain +4 dB. Full validation passed: 555 tests, build, and boundary checks.
